tools/rosbag/src/rosbag/bag.py

```diff
@@ -202,6 +204,7 @@ def initialize(self, bag, gpg_key_user):
         @raise ROSBagException: if GPG key user has already been set
         """
         if bag._mode != 'w':
+            self._gpg_passphrase = os.getenv('ROSBAG_GPG_PASSPHRASE', None)
```

Comment:
Can there be an option to pass it in here, for programmatic use? Eg,

```
def initialize(self, bag, gpg_key_user, passphrase=None):
    ...
    self._gpg_passphrase = passphrase or os.getenv('ROSBAG_GPG_PASSPHRASE', None)
```

Reply:
Updated